setup.py

```diff
@@ -1,8 +1,6 @@
 #! /usr/bin/env python
 
-# Copyright (C) 2008 Cournapeau David <cournape@gmail.com>
-# Copyright (C) 2009 Nathaniel Smith <njs@pobox.com>
-# Copyright (C) 2016 Antony Lee <anntzer.lee@gmail.com>
+# Copyright (C) 2008-2016 The scikit-sparse developers
```

Comment:
If you want to mention them twice I would do it in a consistent way.
```
# Copyright (C) 2008-2016 The scikit-sparse developers:
# Alex Grigorievskiy
# Antony Lee
# Dag Sverre Seljebotn
# Joscha Reimer
# Leon Barrett
# Nathaniel Smith
# Yuri
```
Maybe sorted chronological if this is easy to get from git or in alphabetical order.